samples/sensor/multiple_temp_polling/README.rst

```diff
@@ -0,0 +1,10 @@
+.. _multiple-temp-polling-sample:
+
+Multiple Temp Polling Sample
```

Comment:
ditto zephyr code-sample directive
recommended name for the sample: "Multiple temperature sensors polling"